app/src/main/java/ch/sdp/vibester/activity/LyricsBelongGameActivity.kt

```diff
@@ -201,10 +201,20 @@ class LyricsBelongGameActivity : GameActivity() {
         handler.post(runnable!!)
     }
 
+    /**
+     * Function to set scores in the end of the game
+     */
+    private fun setScores() {
+        if(::gameManager.isInitialized) {
+            super.setScores(gameManager)
+        }
+    }
+
     /** helper functions to test private functions */
```

Comment:
```suggestion
    /* helper functions to test private functions */
    
```
It is a bit weird how the comment is done for the moment.
Either do a comment that describe that below functions are helper
or do a javadoc for each.
But a javadoc that does multiple function at a time does not seems following the conventions.

Reply:
I didn't write this comment nor this part, I just added one * more so it shows green...
I will leave this to the people who do refactor to change.

Reply:
same, will do change myself